wordpress-mobile/WordPress-iOS
# Replace WPNavigationMediaPickerViewController+StatusBarStyle with the preferredStatusBarStyle option

I recently updated the Media Picker to support a new option, `preferredStatusBarStyle` so that we can fix a warning caused by our current behavior of overriding `preferredStatusBarStyle` in a category (see https://github.com/wordpress-mobile/MediaPicker-iOS/pull/327).

This updates WPiOS to move to this.

To test:

- Add some media with the full screen picker. Make sure the status bar is light.

Update release notes:
